tests/test_nam/test_data.py

```diff
@@ -233,5 +233,40 @@ def _t_apply_delay_int(self, n: int, delay: int):
         return x_out, y_out
 
 
+class TestWav(object):
+
+    tolerance = 1e-7
+
+    def test_np_to_wav_to_np(self):
+        # Create random numpy array
+        x = np.random.rand(1000)
+        # Save numpy array as wav file
+        data.np_to_wav(x, "test.wav")
```

Comment:
Nit: Place these files in a place where they will be removed after the test is done e.g. using `with TemporaryDirectory() as tmpdir:`